forui/CHANGELOG.md

```diff
@@ -6,6 +6,16 @@
 
 * Fix style suggestions always displaying actual style name instead of alias.
 
+
+### `FSidebar` (new)
+
+* Add `FSidebar`.
+
+* Add `FSidebarGroup`.
+
+* Add `FSidebarItem`.
```

Comment:
```suggestion
* Add `FSidebar`.
* Add `FSidebarGroup`.
* Add `FSidebarItem`.
```

IMO there isn't a need for the space between lines anymore since we break them down into more manageable sections.